test-e2e/apps/actions.html

```diff
@@ -28,6 +32,8 @@
     const buttons = document.querySelectorAll('.button');
     const body = document.body;
     const select = document.querySelector('#select');
+    const pressCounter = document.querySelector('#pressCounter');
+    let presses = 0;
```

Comment:
type for presses is missed

Reply:
I think here typescript is not supported, it's <script> tag inside html